decidim-budgets/app/assets/javascripts/decidim/budgets/projects.js.es6

```diff
@@ -6,22 +6,22 @@ $(() => {
   const $budgetSummaryTotal = $('.budget-summary__total');
   const $budgetExceedModal = $('#budget-excess');
 
-  const totalBudget = $budgetSummaryTotal.data('total-budget');
+  const totalBudget = parseInt($budgetSummaryTotal.attr('data-total-budget'), 10);
```

Comment:
It turns out that if you modify an actual element's `data-somathing` attribute, it won't affect subsequent `data` calls.

As we're not *destroying* and *creating* elements but *modifying* them instead, this has to be done manually. *sigh*.